Packs/RiskIQDigitalFootprint/IncidentFields/incidentfield-RiskIQ_Asset_AWS_Security_Group_Name.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "incident_riskiqassetawssecuritygroupname",
+    "version": -1,
+    "modified": "2020-07-15T12:00:49.844752893+05:30",
+    "name": "RiskIQ Asset AWS Security Group Name",
+    "ownerOnly": false,
+    "placeholder": "AWS Security Group Name for which the RiskIQAsset of the IP Address type is to be whitelisted.",
+    "description": "AWS Security Group Name for which the RiskIQAsset of the IP Address type is to be whitelisted.",
+    "cliName": "riskiqassetawssecuritygroupname",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "script": "",
+    "fieldCalcScript": "",
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "selectValues": [],
+    "validationRegex": "",
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedTypes": [],
```

Comment:
We did not associate it with any incident type since this will be used for a playbook that is triggered for indicators that will be executed using a job that can have any type of incident.